index.html

```diff
@@ -237,16 +234,26 @@ <h5>Skills</h5>
             <h2 class="form-intro">
                 I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
             </h2>
-            <form action="https://formspree.io/f/mwkzpvyn" method="POST">
-
-                <input type="text" name="name" class="name" placeholder="Full name" maxlength="30" required>
-
-                <input type="email" name="email" class="email" placeholder="Email address" required>
+            <form id="form-validation" action="https://formspree.io/f/mwkzpvyn" method="POST">
+                <ul>
+                    <li>
+                <input type="text" name="name" class="name" placeholder="Full name" maxlength="30">
+                <small class="error-msg"></small>
+                    </li>
+                    <li>
+                <input type="email" name="email" class="email" placeholder="Email address">
+                <small class="error-msg"></small>
```

Comment:
- [ ] Following the project requirements, the error message should show up close to the "submit" button.